site-content/source/modules/ROOT/pages/development/documentation.adoc

```diff
@@ -9,17 +9,16 @@ The official Cassandra documentation lives in the project's git
 repository.
 We use a static site generator, http://www.antora.org/[Antora], to create pages hosted at
 link:/doc/latest/[cassandra.apache.org].
-
```

Comment:
We need this blank line restored. (i.e. please don't remove it)

Because removing it makes it look like…

<img width="971" alt="Screenshot 2022-08-14 at 13 24 34" src="https://user-images.githubusercontent.com/559444/184534506-97f72802-be9f-4af8-8620-6114dba4ebd1.png">

while originally it looks like (which is how we want it)

<img width="971" alt="Screenshot 2022-08-14 at 13 26 09" src="https://user-images.githubusercontent.com/559444/184534548-da0b227f-0682-45e2-b310-9c270d258b1b.png">